### ISSUE TITLE
Better scheduling algorithm

### ISSUE THREAD

**Issue by kanav99** [SciML/DiffEqBenchmarkServer.jl#5]
This bot may be used for testing purposes later, so we might need a new scheduling algorithm for the bot. 
1. We can run simultaneous tests.
2. But the benchmark job should always run on the idle runner.
3. Support multiple runners.

**Comment by ChrisRackauckas** [comment]
I think what's more important is to make sure that we can trigger jobs with a different tag on a different machine. We are getting a second computer setup on Thursday which will have a GPU, but not a great CPU. So we want to be able to set some scripts to run with just the diffeqgpu tag and use that runner.

**Comment by kanav99** [comment]
Yeah we should do this on a priority basis. We can use Gitlab Runner tags (we don't use them right now) and develop an argument `runbenchmarks(on = gpu)` so that bot knows where we should run that.

**Comment by ChrisRackauckas** [comment]
For benchmarks, does it run both commits each time, or cache master? If it caches master, we need to make sure each script has a dedicated machine.

**Comment by kanav99** [comment]
No, right now it runs both the branches.

**Comment by ChrisRackauckas** [comment]
Yeah and that's probably better. So okay yes a priority system is all that's necessary.
